src/main/java/org/scalameter/japi/annotation/teardownAfterAll.java

```diff
@@ -0,0 +1,18 @@
+package org.scalameter.japi.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/** Optional annotation that points to a
```

Comment:
Optional annotation that points to a teardown-after-all method -- the method in the 
    same class that gets invoked after all the benchmarks complete.